src/Elliptic/Systems/Poisson/Equations.hpp

```diff
@@ -0,0 +1,168 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/FaceNormal.hpp"
+#include "Domain/Tags.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/ContainerHelpers.hpp"
+#include "Utilities/TMPL.hpp"
+
+class DataVector;
+template <size_t>
+class Mesh;
+
+namespace Tags {
+template <typename>
+struct Normalized;
+}  // namespace Tags
+
+namespace LinearSolver {
+namespace Tags {
+template <typename>
+struct Operand;
+}  // namespace Tags
+}  // namespace LinearSolver
+
+namespace Poisson {
+struct Field;
+template <size_t>
+struct AuxiliaryField;
+}  // namespace Poisson
+
+namespace Poisson {
+
+template <size_t Dim>
+struct ComputeMixedVolume {
```

Comment:
why is Mixed in the name?

Reply:
This computes the volume contribution for the mixed system (I added this to the documentation). For the primal system the volume contribution is different (just the laplacian).